sympy/ntheory/tests/test_digits.py

```diff
@@ -0,0 +1,109 @@
+import os
+import random
+
+random.seed(os.urandom(10 ** 6))
+
+from itertools import chain
+from unittest import TestCase
+
+from sympy.ntheory.digits import is_pandigital
+
+
+class TestDigits(TestCase):
```

Comment:
We are not using `unittest` for now. You should write the tests in pytest format.

Reply:
Thanks, I've done that now.